CustomScript/HandlerManifest.json

```diff
@@ -1,13 +1,17 @@
-[{
-  "name": "CustomScript",
-  "version": "1.5.1.0",
-  "handlerManifest": {
-    "installCommand": "customscript.py -install",
-    "uninstallCommand": "customscript.py -uninstall",
-    "updateCommand": "customscript.py -update",
-    "enableCommand": "customscript.py -enable",
-    "disableCommand": "customscript.py -disable",
-    "rebootAfterInstall": false,
-    "reportHeartbeat": false
+[
+  {
+    "namespace": "Microsoft.OSTCExtensions",
+    "name": "CustomScriptForLinux",
+    "extension_version": "1.5.2.0",
+    "version": 1,
```

Comment:
please make it a `1.0` explicitly, you may lose 2 bytes but you may end up saving backwards compatibility or something like that when waagent changes one day and decides to look at this field. everything is python so this is prolly gonna deserialize into an int rather than float.